src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java

```diff
@@ -15,18 +11,35 @@
 import org.testng.annotations.Test;
 import org.testng.asserts.SoftAssert;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class AddClubPopUpWithManagerTest extends LoginWithManagerTestRunner {
 
-    private static final String VALID_CLUB_NAME = "Add club name";
+    private static final String VALID_CLUB_NAME = "Add club name 2";
```

Comment:
```suggestion
    private static final String VALID_CLUB_NAME = "Add club name";
```

Why did you change the value of const?
If you want you can do your own temporary variable inside the test